### PR TITLE
Add static helpers

### LINKED ISSUE #2795: Add web.static and web.RouteTableDef.static
As aliases for UrlDispatcher.add_static.

The idea is adding static file resources in declarative way: `app.add_routes([web.static(...)])` and `table.static(...)`.

Part of #2766

### PR DESCRIPTION
**Pull request by asvetlov**
Fixes #2795

### CODE REVIEW

File: tests/test_route_def.py [hunk 1]
```diff
@@ -256,6 +272,17 @@ def test_repr_route_def_with_extra_info():
     assert repr(rd) == "<RouteDef GET /path -> 'handler', extra='info'>"
 
 
+def test_repr_static_def():
+    routes = web.RouteTableDef()
+
+    routes.static('/prefix', '/path', name='name')
```
Comment (kxepal) [inline tests/test_route_def.py:278]: `@`

Comment (asvetlov) [reply tests/test_route_def.py:278]: No, this method is intentionally not a decorator: static has no user-supported handler, only `prefix`, `path` and optional parameters. I'll document it properly.

The reason why `url.static()` and `routes.static` should exist is it is a part of configuration which very often is constant (safe to be done at module level). No need to add `app.add_statiic()` shortcut.

`app.router.add_static()` is redundant (but still exist and do all work internally).
`app.add_subapp` is a different beast: it requires `subapp` instance, that's why I don't want to add shourcuts for `add_subapp`.

Comment (kxepal) [reply tests/test_route_def.py:278]: Aha, I see. Now it looks fine. On previous version there was a function right after and that looks like missed @ from decorator.

**Comment by asvetlov** [comment]
Thanks @kxepal

**Pull request merged**